test/e2e/install_test.go

```diff
@@ -390,25 +389,8 @@ func getArtifactsOutput(t *testing.T) {
 		}
 	}
 
-	// Get all Bundles in the namespace and save them to the artifact path.
-	bundles := rukpakv1alpha1.BundleList{}
-	if err := c.List(context.Background(), &bundles, client.InNamespace("")); err != nil {
-		fmt.Printf("Failed to list bundles: %v", err)
-	}
-	for _, bundle := range bundles.Items {
-		// Save bundle to artifact path
-		bundleYaml, err := yaml.Marshal(bundle)
-		if err != nil {
-			fmt.Printf("Failed to marshal bundle: %v", err)
-			continue
-		}
-		if err := os.WriteFile(filepath.Join(artifactPath, bundle.Name+"-bundle.yaml"), bundleYaml, 0600); err != nil {
-			fmt.Printf("Failed to write bundle to file: %v", err)
-		}
-	}
-
```

Comment:
Could you pleae explain why we are removing this?

Reply:
There is no such thing as the Bundle API anymore.